### PR TITLE
Add missing fix return type code action in commit action context

### LINKED ISSUE #37287: Change return type code action not working in commit statement
**Description:**

https://user-images.githubusercontent.com/46120162/183309739-c42f0213-7dbc-46da-83eb-81173f6e2acd.mov

**Steps to reproduce:**
```
function test() {
    transaction {
        doSomething();
        check commit;
    }
}

function doSomething() {

}
```
**Affected Versions:**
2201.2.0

### PR DESCRIPTION
**Pull request by malinthar**
## Purpose
$subject

Fixes #37287

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/FixReturnTypeTest.java [hunk 1]
```diff
@@ -46,13 +46,13 @@ public void test(String config) throws IOException, WorkspaceDocumentException {
...
                 {"fixReturnType1.json"},
-//                {"fixReturnType2.json"},
+//                {"fixReturnType2.json"}, #36410
                 {"fixReturnType3.json"},
                 {"fixReturnType4.json"},
                 {"fixReturnType5.json"},
                 {"fixReturnTypeWithImports1.json"},
                 {"fixReturnTypeWithClass1.json"},
-//                {"fixReturnTypeWithClass2.json"},
+//                {"fixReturnTypeWithClass2.json"},
```
Comment (mindula) [inline language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/FixReturnTypeTest.java:55]:
```suggestion
//                {"fixReturnTypeWithClass2.json"},
```

**Comment by IMS94** [comment]
@malinthar shall we address the review comments

**Review by gayalkuruppu — APPROVED** [review]
LGTM

**Review by TharushiJay — APPROVED** [review]
LGTM!

**Pull request merged**